scripts.js

```diff
@@ -1,3 +1,7 @@
+$(function(){
+	console.log('DOM loaded - you can have fun');
```

Comment:
ok, but your code should be done after DOM is loaded. so you need to put your code inside function that check is DOM loaded